---
title: "The i5k Webinar Series"
type: single
permalink: /webinar
---

## Monthly webinars start October 5th at 11am EDT

Worried about error rates, validation issues, sample sizes or other details of genomic sequencing, assembly and analysis?  Bring your questions and help us have a lively discussion with our speakers.

__Who:__  Webinars are open to all researchers interested in arthropod genomics.

__What:__  Presentations by representatives from genomic technology companies, databases and research groups.  Webinars will be 1 hour long, but presentations will be kept short enough to allow plenty of time for participants to ask questions.

__When:__  First Wednesday of each month at 11am EDT

__Why:__  Some of the impacts made by i5k are through training and education that foster open and active participation in the establishment and adoption of best practices for methodologies, informatics and standards.  Our goal for the webinar series is to provide outreach and education by introducing researchers to a variety of techniques, technologies and best practices to help move their research forward.

##### To get news about the webinar series and other i5k activities, please subscribe to the Arthropod News [mailing list](https://listserv.ksu.edu/?SUBED1=ARTHROPODNEWS&A=1)


Be an active attendee!  Worried about error rates, validation issues, sample sizes or other details?  Bring your questions and help us have a lively discussion with our speakers.


## More information to come, here is a peak at the schedule:

### October 5th
__Speaker:__ Jonas Korlach, Chief Scientific Officer, and Sarah Kingan, Senior Scientist Bioinformatics, Pacific Biosciences<br>
__Title:__ High-Quality De Novo Insect Genome Assemblies using PacBio Sequencing<br>
__Summary:__ PacBio Sequencing is characterized by very long sequence reads (average >10,000 bases), lack of GC-bias, and high consensus accuracy. These features have allowed the method to provide a new gold standard in de novo genome assemblies, producing highly contiguous (contig N50 > 1 Mb) and accurate (> QV 50) genome assemblies. We will briefly describe the technology and then highlight the full workflow, from sample preparation through sequencing to data analysis, on examples of insect genome assemblies, and illustrate the difference these high-quality genomes represent with regard to biological insights, compared to fragmented draft assemblies generated by short-read sequencing.


### November 2nd
__Speaker:__ 10x Genomics


### December 7th
__Speaker:__ Dr. Ivan Liachko, CEO/CSO, Phase Genomics Inc., Seattle, WA; Dept. of Genome Sciences, University of Washington, Seattle, WA<br>
__Title:__ A rapid method for end-to-end genome assembly, pathogen discovery, and metagenomic deconvolution<br>
__Summary:__ I will describe the Hi-C method and its adaptation toward scaffolding of genomes and metagenomic deconvolution. I will give examples of our work scaffolding large genomes (plant/animal/fungal) as well as assembling prokaryotic and eukaryotic genomes from mixed microbial communities.

###January 4th
__Speaker:__ Dr. Scott Emrich, UND Director of Bioinformatics; Depts. of Computer Science and Engineering and Biological Sciences (concurrent), University of Notre Dame; contact PI, NIH/NIAID VectorBase, a Bioinformatics Resource Center (BRC)

###February 1st
__Speaker:__ Prof. Evgeny M. Zdobnov and Dr Robert M. Waterhouse, University of Geneva Medical School & Swiss Institute of Bioinformatics, Geneva, Switzerland<br>
__Title:__ OrthoDB: an evolutionary perspective to interpreting genomics data<br>
__Summary:__ Orthology is a cornerstone of comparative genomics, and such approaches are well-established as immensely valuable for gene discovery and characterization, offering evolutionarily-qualified hypotheses on gene function by identifying “equivalent” genes in different species.<br>
The OrthoDB hierarchical catalogue of orthologues represents a comprehensive resource of comparative genomics data that delineates the evolutionary histories of millions of genes from thousands of species.<br>
OrthoDB resources and tools enable extensive orthology-based genome annotation and interpretation in a comparative genomics framework that incorporates the growing numbers of sequenced genomes.

### March 1st
__Speaker:__ Dr. Jack Werren

### April 5th
__Speaker:__ Dr. Keith Hopper, Research Entomologist, Beneficial Insect Introductions Research Unit, USDA-ARS, Newark, DE

### May 3rd
__Speaker:__ Dr. Terence Murphy, Staff Scientist, Gene Project Lead, NCBI/NLM/NIH<br>
__Title:__ The in’s and out’s of genome assembly submission and annotation at NCBI

### June 7th
__Speaker:__ BioNano


