---
title: "The i5k Webinar Series"
type: single
permalink: /webinar
---
<br>
__[Details](#details)__<br>
__[Current Schedule](#current)__<br>
__[Previous Presentations](#previous)__<br>


### To get news about the webinar series and other i5k activities, please subscribe to the low E-mail volume Arthropod News [mailing list](https://listserv.ksu.edu/?SUBED1=ARTHROPODNEWS&A=1)<br><br>

<a name="details"></a> 
Worried about error rates, validation issues, sample sizes or other details of genomic sequencing, assembly and analysis?  Bring your questions and help us have a lively discussion with our speakers.

__Who:__  Webinars are open to all researchers interested in arthropod genomics.

__What:__  Presentations by representatives from genomic technology companies, databases and research groups.  Webinars will be 1 hour long, but presentations will be kept short enough to allow plenty of time for participants to ask questions.

__When:__  First Wednesday of each month from 11am to 12 noon EDT<br>*Event times vary relative to your timezone. Please click the date in the schedule below to confirm the time in your location.*

__Why:__  Some of the impacts made by i5k are through training and education to promote the adoption of best practices for methodologies, informatics and standards.  Our goal for the webinar series is to provide outreach and education by introducing researchers to a variety of techniques, technologies and best practices to help move their research forward.

__How:__  The webinars will use the AT&T Connect software
<br>*We will have 250 connection lines available so please consider gathering in groups where possible to access the webinar.*

| __Connection Type__ | __Details__ |
| ---- | ---- |
|Web Conference Connection|This connection will allow you to view the video.  You may join your audio through your computer or if your computer does not support an audio connection, you may connect the audio portion of the webinar via a telephone (see Audio Connection type for details).<br><br>1. [Connect to the Web Conference using the AT&T Connect application](https://connect16.uc.att.com/usda/meet/?ExEventID=86042415)<br>Link will provide a prompt to download the application and will launch the application once installed.<br><br>2. [Connect through your WEB BROWSER without downloading the AT&T Connect application](https://www.connectmeeting.att.com)<br>Meeting Number:  8888449904 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Meeting Code: 5909637 |
|Audio Connection via Telephone|1. USA Access:  USA Toll-Free: 888-844-9904, then enter Code: 5909637# or USA Caller Paid: 816-423-4261, then enter Code: 5909637# <br><br>2. [International Access Numbers](https://www.teleconference.att.com/servlet/glbAccess?process=1&accessNumber=8888449904&accessCode=5909637)<br> A number in your country or in a country close to you (may not be toll free).<br><br>3. When prompted, enter the Meeting Access Code: 5909637# <br><br>4. Link the computer session with the telephone session:  After you have both the computer and telephone sessions established, click on the top menu tab "Audio/Video."  Select the second option "Telephone Connection Instructions."  Look for the section "Already Connected by Phone."  Locate the "Link" number and type the link number directly into the keypad of the telephone as #-sign, the six digit number, and follow with the #-sign.  Close the dialog box.<br>*It is important to link your computer and telephone sessions so only one connection line is used per participant.*|

 

<a name="current"></a>

## Current Schedule

| __Date__ | __Talk Information__|
| ---- | ---- |
|[November&nbsp;1st](https://www.timeanddate.com/worldclock/fixedtime.html?msg=i5k+webinar&iso=20171101T11&p1=263&ah=1)|<br>__Speaker:__ Jonathan Coddington, Director, Global Genome Initiative and Senior Scientist, National Museum of Natural History, Smithsonian Institution<br><br>__Title:__ Not Too Much Life: the Global Genome Initiative and the Global Genome Biodiversity Network<br><br>__Summary:__ How much life is there and how is it structured? What strategy will best "illuminate" the dark spaces of genomic knowledge across the tree of life? What progress have we made? What resources are rate-limiting? In  particular, how can collections-based institutions help i5k? Bioinformatics contributions...<br><br>Will slides be released: Yes<br>Will the presentation be recorded: Yes<br><br>|

<a name="previous"></a>

## Previous Presentations


| __Date__&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | __Talk Information__|
|---- |-----|
|October&nbsp;5,&nbsp;2016 |<br>__Speaker:__ Jonas Korlach, Chief Scientific Officer, and Sarah Kingan, Senior Scientist Bioinformatics, Pacific Biosciences<br><br>__Title:__ High-Quality De Novo Insect Genome Assemblies using PacBio Sequencing<br><br>__Summary:__ PacBio Sequencing is characterized by very long sequence reads (average >10,000 bases), lack of GC-bias, and high consensus accuracy. These features have allowed the method to provide a new gold standard in de novo genome assemblies, producing highly contiguous (contig N50 > 1 Mb) and accurate (> QV 50) genome assemblies. We will briefly describe the technology and then highlight the full workflow, from sample preparation through sequencing to data analysis, on examples of insect genome assemblies, and illustrate the difference these high-quality genomes represent with regard to biological insights, compared to fragmented draft assemblies generated by short-read sequencing.<br><br>__[Slides](webinar_slides/i5K_webinar_PacBio_Oct5_2016.pdf)__<br>__[Video recording](https://www.youtube.com/watch?v=vAMAc4gGIlQ)__<br><br>|
|November&nbsp;2,&nbsp;2016 |<br>__Speaker:__ Dr. Anup Parikh, Director of Product Marketing, 10x Genomics<br><br>__Title:__ De Novo assembly with 10x Genomics <br><br>__Summary:__ Discuss limitations in the current ways we approach genome analysis.  Specifically discuss the state of de-novo assembly for genome analysis and demonstrate a straightforward and low-cost method for creating true diploid de novo assemblies with 10x Genomics.<br><br>__[Slides](webinar_slides/i5k_10x_Genomics-2016.11.02.pdf)__<br>__[Video Recording](https://youtu.be/Yl1_WX213LI)__<br><br>|
|December&nbsp;7,&nbsp;2016|<br>__Speaker:__ Dr. Ivan Liachko, CEO/CSO, Phase Genomics Inc., Seattle, WA; Dept. of Genome Sciences, University of Washington, Seattle, WA<br><br>__Title:__ A rapid method for end-to-end genome assembly, pathogen discovery, and metagenomic deconvolution<br><br>__Summary:__ I will describe the Hi-C method and its adaptation toward scaffolding of genomes and metagenomic deconvolution. I will give examples of our work scaffolding large genomes (plant/animal/fungal) as well as assembling prokaryotic and eukaryotic genomes from mixed microbial communities.<br><br>__[Video Recording](https://www.youtube.com/watch?v=FmElQwcyut4&t=5s)__<br><br>|
|January&nbsp;4,&nbsp;2017|<br>__Speaker:__ Dr. Scott Emrich, UND Director of Bioinformatics; Depts. of Computer Science and Engineering and Biological Sciences (concurrent), University of Notre Dame; contact PI, NIH/NIAID VectorBase, a Bioinformatics Resource Center (BRC)<br><br>__Title:__ Non-model arthropod assembly:  past, present and future<br><br>__Summary:__ Over 40 important arthropod vectors have been sequenced and placed into VectorBase.  I will present general principles of genome assembly and analysis ranging from (inbred) colonies to more recent work on field-sourced individuals.  Based on these multiple consortium efforts, I will provide information on how to take full advantage of informatics resources available, including but not limited to VectorBase and overcoming potential road-blocks finding or submitting non-model genome data.  Finally, I will share emerging results efforts, including my own, to improve genome assemblies of important mosquito vectors using data discussed in other i5K webinars in this series.<br><br>__[Slides](webinar_slides/i5k_webinar_VectorBase_01_04_2017.pdf)__<br>__[Video recording](https://www.youtube.com/watch?v=ylXuIjD0dvE)__<br><br>|
|February&nbsp;1,&nbsp;2017|<br>__Speaker:__ Prof. Evgeny M. Zdobnov and Dr Robert M. Waterhouse, University of Geneva Medical School & Swiss Institute of Bioinformatics, Geneva, Switzerland<br><br>__Title:__ OrthoDB: an evolutionary perspective to interpreting genomics data<br><br>__Summary:__ Orthology is a cornerstone of comparative genomics, and such approaches are well-established as immensely valuable for gene discovery and characterization, offering evolutionarily-qualified hypotheses on gene function by identifying “equivalent” genes in different species.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The OrthoDB hierarchical catalogue of orthologues represents a comprehensive resource of comparative genomics data that delineates the evolutionary histories of millions of genes from thousands of species.<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;OrthoDB resources and tools enable extensive orthology-based genome annotation and interpretation in a comparative genomics framework that incorporates the growing numbers of sequenced genomes.<br><br>__[Slides](webinar_slides/i5K_webinar_OrthoDBFeb_01_2017.pdf)__<br>__[Video Recording](https://www.youtube.com/watch?v=cswNRFLMTN4)__<br><br>|
|March&nbsp;1,&nbsp;2017|<br>__Speaker:__ Dr. Sven Bocklandt, Senior Application Specialist, Bionano Genomics<br><br>__Title:__ Improve genome accuracy and contiguity using Bionano Next-Generation Mapping<br><br>__Summary:__ Generating high quality finished genomes remains challenging. Many genomes are highly repetitive, and NGS has led to incomplete assemblies that contain large numbers of contigs and limited long-range information. <br>Bionano images extremely long molecules, from 150 kbp to megabase pairs in length, to reveal the true long-range structure of the genome. We will discuss how Bionano’s de novo genome maps can increase the contiguity of assemblies up to 100-fold over NGS alone. Because it is the only non-sequencing based scaffolding method, it can error correct assemblies. We will also discuss the difficulties and recent progress in detecting large structural variation.<br><br>__[Slides](/webinar_slides/i5K_webinar_Bionano_Mar_01_2017.pdf)__<br>__[Video Recording](https://www.youtube.com/watch?v=rSYHJMqtJmc)__<br><br>|
|April&nbsp;5,&nbsp;2017|<br>__Speaker:__ Dr. Keith R. Hopper, Research Entomologist, Beneficial Insect Introductions Research Unit, USDA-ARS, Newark, Delaware<br><br>__Title:__ Phylogenetics and quantitative genetics of host specificity in aphid parasitoids in the genus Aphelinus<br><br>__Summary:__ Differences in parasitism success among potential host species can provide strong selection for divergence and speciation in parasitic Hymenoptera. Here we report research on the genomics and genetics of host specificity in Aphelinus species. We have sequenced, assembled, and annotated the genomes and transcriptomes of >10 Aphelinus species.  Using coding sequences, we developed a robust phylogeny, onto which we mapped parasitism of diverse species of aphids. For some aphid species, parasitism was phylogenetically conserved, with closely related parasitoids showing similar levels of parasitism. For other aphid species, parasitism diverged between closely related parasitoids, consistent with host-driven speciation. To explore the genetic architecture of differences in host specificity, we crossed and backcrossed A. atriplicis, which readily parasitizes Diuraphis noxia, with A. certus, which rarely parasitizes this aphid. Using genetic markers from reduced-representation genomic libraries, we mapped quantitative trait loci (QTL) affecting parasitism of D. noxia. We found eight QTL (six of which interacted in their effects) that explained 39% of the variation in parasitism D. noxia among backcross females. To help identify candidate genes, we compared the genomes and transcriptomes of these parasitoid species to find proteins that diverged in sequence or expression, and we tested whether these divergent loci mapped to QTL affecting parasitism of D. noxia. So far, we have found 15 divergent genes that mapped to parasitism QTL or significantly affected parasitism by themselves. These are among the first results on the genetic architecture of host specificity in parasitic wasps.<br><br>__[Slides](/webinar_slides/i5K_webinar_K_Hopper_Apr_05_2017.pdf)__<br>__[Video Recording](https://www.youtube.com/watch?v=pT5BIHG2ekg)__<br><br>|
|May&nbsp;3,&nbsp;2017|<br>__Speaker:__ Dr. Kevin Hackett, i5k Co-chair<br><br>__Summary:__ Discussion of the Earth BioGenome Project (EBP) including: 1) How getting folks to identify with i5K and EBP can lift all boats; 2) Why we need to invest in genomics infrastructure such as i5K and EBP, in contrast to spending all funding in hypothesis-driven research. <br><br>|
|June&nbsp;7,&nbsp;2017|<br>Wonderful to see everyone at the [10th Annual Arthropod Genomics Symposium and VectorBase Workshop](http://globalhealth.nd.edu/10th-annual-arthropod-genomics-symposium/), which was held from June 7 - June 10, 2017 at the University of Notre Dame.<br><br>|
|July&nbsp;5,&nbsp;2017|<br>__Speaker:__ Gerard Coyne, Senior Technical Applications Specialist, Oxford Nanopore Technologies<br><br>__Title:__ Real time DNA sequencing using Oxford Nanopore Technologies 'nanopore sensing' platform<br><br>__Summary:__ Oxford Nanopore Technologies has developed a disruptive platform for the direct, electronic analysis of single molecules.  Our instruments the MinION (TM) and the PromethION (TM) are adaptable for the detection and analysis of a range of analytes that include DNA, RNA, proteins and small molecules.  At the heart of our platform is a biological protein called a 'nanopore'.  A single nanopore create a hole in a membrane made from a proprietary synthetic polymer.  An electric potential is applied across the membrane resulting in a current flowing only through the aperture of the nanopore.  Single molecules that enter the nanopore cause characteristic disruptions in the current, by measuring these disruptions single molecules from a sample are identified.  The MinION is a small device that is designed for portability and simplicity of its workflow.  The MinION plugs into a standard PC or laptop using the USB port.  The PromethION is a standalone high throughput benchtop instrument that provides the flexibility to run 100's of samples in an asynchronous manner.  This allows for large projects that requires the flexibility and throughput to interrogate complex eukaryotic genomes.  Oxford Nanopore is integrating the data produced by the MinION and PromethION into a cloud-based analytics company, Metrichor.  Metrichor is powered by its EPI2ME platform.  Metrichor is providing tools to automate data analysis workflows to help people track, trend and predict biological data resulting in real time actionable interpretation of their data.  Users of the technology have access to our 'Nanopore Community’.  The Nanopore Community helps new users get started with technical documentation as well as user driven forums and encourages discussion and collaborative experimentation using our technology.  There is a growing list of publications on the many uses for our nanopore sensing platform that include field based applications, real time pathogen detection and surveillance, metagenomics analysis, anti-microbial resistance detection, education and many more including sequencing on the International Space Station.<br><br>No slides or presentation recording made available.|
|August&nbsp;2nd,&nbsp;2017 |<br>__Speaker:__ Brandon Rice, Head of Development and Strategy, Dovetail Genomics <br><br>__Title:__ Heaps of Chromosomes, New Scales and Evolving Paradigms in Genome Assembly<br><br>__Summary:__ The field of genomics has arrived at an inflection point in its history for its most fundamental resource: genome assemblies. Genome assembly is increasingly stream-lined and the results of increasingly outstanding quality, with chromosome-scale assemblies as the new standard. At Dovetail we aim to drive genome assemblies to full commoditization; focus on your science, not on the vagaries of sequencing technologies and analysis approaches. Come hear about the tool-kit and processes we have assembled and those we’re developing to increase focus on science over assembly as the focus of genomics for every organism.<br><br>__Bonus Presentation from User Perspective__ <br>__Speaker:__ Dr. Brenda Oppert, Research Molecular Biologist, Center for Grain and Animal Health Research, Stored Product Insect and Engineering Research Unit, USDA-ARS, Manhattan, Kansas<br><br>__Title:__ Reading through the repeats: Dovetail technology improves assembly of insect genomes<br><br>__[Slides](webinar_slides/i5k_webinar_Dovetail_08_02_2017.pdf)__<br>__[Video Recording](https://www.youtube.com/watch?v=TneDnO_doOw)__<br><br>|
|September&nbsp;6th|<br>__CANCELED__ <br>__Speaker:__ Illumina’s Applied Genomics group<br><br>|
|October&nbsp;4th|<br>__Speaker:__ Monica Poelchau and Christopher Childers, USDA-ARS-NAL<br><br>__Title:__ The i5k Workspace@NAL: a pan-Arthropoda genome database<br><br>__Summary:__ The i5k Workspace@NAL mission is to support any arthropod genome project by supplying tools and resources to help further applied genomics research.  Each genome project has search, gene annotation and visualization tools.  We will give an introduction to the workspace, highlighting our resources and services with an emphasis on some of our newer features, such as the new system for starting a project and uploading data and the post annotation QC and Official Gene Set creation pipeline.<br><br>[Slides](webinar_slides/i5k_webinar-i5k_workspace_Oct04-2017.pdf)<br>[Video Recording](https://youtu.be/HYo2RQa4BUI)<br><br>|
